device.py

```diff
@@ -44,8 +96,72 @@ def identify(self, uuid):
         }
         return self.request('/blink', 'POST', data=data)
 
-    def get_name(self, uuid):
-        return self.get(uuid)['name']
+    def rename(self, uuid, new_name):
+        if self.has(uuid):
+            params = {
+                'filter': 'uuid',
+                'eq': uuid
+            }
+            data = {
+                'name': new_name
+            }
+            return self.request('device', 'PATCH', params=params, data=data)
+        else:
+            print(Message.NO_DEVICE_FOUND.format(value=uuid,dev_att="uuid"))
```

Comment:
Again, this method is not clear what it returns, or it returns many different things depending on circumstances. We should generally be consistent with how we handle API calls and what values we return so that a user of our library doesn't have to worry and mess with ``if this: blah() else: blablah()``